features/crud.feature

```diff
@@ -16,7 +16,9 @@ Feature: Create-Retrieve-Update-Delete
           "value1",
           "value2"
         ]
-      }
+      },
+      "dummyFloat": 1.2,
+      "dummyPrice": 1.2
```

Comment:
Note that I think the second solution would be cleaner, as that there are no difference between floats and decimals in json.